challenges/2021_cloud_base_height/02_explore_transformed_data.ipynb

```diff
@@ -0,0 +1,1532 @@
+{
```

Comment:
<p>Should this still be in the final version of the notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I feel that the package reload is important to any development or discovery on the code since it can be costly to restart the kernel after making a small update to the other files. I have disabled it in the notebook settings for the git repo, but kept in the cell to help anyone who wants to play around with the notebook